### PR TITLE
Update pr template

### PR DESCRIPTION
**Pull request by mie-h**
## I'm submitting a ...

- [ ] bug report
- [x] feature request

### CODE REVIEW

**Review by huyfififi — APPROVED** [review]
Maybe we better contain only objective ones. `code is easy to understand` cannot be measured and vary from engineers. but I may be nitpicking.

I'm just curious, where did you get the checklist? 😃

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 ## What is the motivation / use case for changing the behavior?
+
+## Checklist
+
+- [ ] code is easy to understand
+- [ ] tests are present (if necessary)
+- [ ] no commented out code
+- [ ] no debbugging code
+- [ ] errors are handled
+- [ ] passed linter
```
Comment (huyfififi) [inline .github/pull_request_template.md:19]: `nit` maybe we already run `ruff` as a workflow automatically, so we can be sure that it's done?

**Comment by mie-h** [comment]
> Maybe we better contain only objective ones. `code is easy to understand` cannot be measured and vary from engineers. but I may be nitpicking 👍
> 
> I'm just curious, where did you get the checklist? 😃

https://dev.to/miledi_delafer/pull-request-template-444
I got it from here and modified it on top of it.

**Review by huyfififi — APPROVED** [review]
👍

**Review by LucileTech — APPROVED** [review]

**Pull request merged**